tests/test_show_version.py

```diff
@@ -0,0 +1,111 @@
+import builtins
+import os
+import pathlib
+
+import pytest
+from unittest.mock import mock_open
+from unittest.mock import patch
+
+from komodo.show_version import read_config
+from komodo.show_version import guess_release
+from komodo.show_version import guess_komodo_path
+from komodo.show_version import guess_komodoenv_path
+from komodo.show_version import get_version
+
+
+@pytest.fixture()
+def mock_komodo_env_vars():
+    """
+    Provide the environment vars from a 'vanilla' komodo environment.
+    """
+    env = {
+        "PATH": "/foo/bar/komodo-release-0.0.1/root/bin",
+        "KOMODO_RELEASE": "komodo-release-0.0.1",
+    }
+    with patch.dict(os.environ, env):
+        yield
+
+
+@pytest.fixture()
+def mock_komodoenv_env_vars():
+    """
+    Provide the 'fake' environment vars from a komodoenv environment.
+    """
+    env = {
+        "PATH": "/quux/komodo-release/root/bin",
+        "KOMODO_RELEASE": "/quux/komodo-release",
```

Comment:
I suppose this test covers the case in https://github.com/equinor/komodo-releases/pull/3043/files#diff-15f0a4593f5be26df91be4cafda7792e49d2c795c2b82ef1e72b58bd8abf89b1R30-R31

Reply:
:thinking: I'll ask you more about this when I see you.

Reply:
I was specifically looking for a test that covers the case when the env variable `$KOMODO_RELEASE` contains the full path and not just a string, since I used that as a workaround in the link above. And it looks like you have covered it 👍🏻

Reply:
I think it does.